compositions/upbound-aws-provider/sqs/sqs.yaml

```diff
@@ -43,7 +43,6 @@ spec:
           forProvider:
             delaySeconds: 10
             messageRetentionSeconds: 345600
-            kmsMasterKeyId: alias/aws/sqs
```

Comment:
This removes encryption at transit and is really not recommended by AWS.  We should capture best practices when we can.

Reply:
This does not remove the encryption. The alias is created dynamically and passed in the patch. This removes overriding the key with hard-coded value.

Reply:
I moved the default value to the definition.